src/game/logic/object/update/laserupdate.cpp

```diff
@@ -310,9 +310,9 @@ void LaserUpdate::Update_Start_Pos()
                 m_startPos.x = m.Get_X_Translation();
                 m_startPos.y = m.Get_Y_Translation();
                 m_startPos.z = m.Get_Z_Translation();
+            } else {
+                m_startPos = *drawable->Get_Position();
             }
-        } else {
-            m_startPos = *drawable->Get_Position();
         }
 
         if (!(m_startPos == oldpos)) {

```

Comment:
Move this if statement inside the if (drawable) as well. Looks good otherwise.

Reply:
ok